### PR TITLE
Change "database" to "catalog"

### PR DESCRIPTION
**Pull request by wy8881**
Fixes koppor/jabref#615 

Rename "database" to "catalog" for ul and I10n

Change the corresponding translation for some languages

### Mandatory checks
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Hi you only need to edit the English translations, the rest is done via crowdin https://devdocs.jabref.org/code-howtos/localization.html

**Comment by wy8881** [comment]
> Hi you only need to edit the English translations, the rest is done via crowdin https://devdocs.jabref.org/code-howtos/localization.html

Hello, thanks for your advice. I checked this website and I am confused. It seems like I need to use the function Localization.lang() when there needs to be translated, but I can't find this function be used in ManageStudyDefinition.fxml.
I'm not sure whether it's fine just add a key-value pair in JabRef_en.properties and I'll try to run the test again. Sorry for not checking related documents in advance. I'm totally new to making contributions to an open-source project.

**Comment by Siedlerchr** [comment]
Thanks, we will add a section on fxml, in the fxml files it's simply a % sign in front of the string

**Comment by wy8881** [comment]
Thanks for explaining it, I googled the sign % before and found nothing.

**Review by Siedlerchr — APPROVED** [review]

**Comment by calixtus** [comment]
Hi, thanks for your interest in JabRef. Please not, that as long as you are not a native speaker, please remove all the language changes and leave it to crowdin to translate!

Also I think the maintainers should discuss such ui changes at least in the devcall. This is maybe some fundamental discussion, if every issue in the jabref or koppor repo including an ui change is already meant to be implemented without being talked about. In this case, as this is only a very small change in the SLR subsystem, it's probably not a big deal. But nevertheless it's something to be talked about.

File: src/main/resources/l10n/JabRef_ko.properties
Comment (calixtus) [inline src/main/resources/l10n/JabRef_ko.properties:1852]: Does this have something to do with your code changes?

**Review by calixtus — CHANGES_REQUESTED** [review]

File: src/main/resources/l10n/JabRef_ko.properties
Comment (wy8881) [reply src/main/resources/l10n/JabRef_ko.properties:1852]: Hi, I have already deleted all the translations in the non-English file. Sorry for that, next time I definitely communicate with others first. I'm not sure what is the reason for labelling the Filename format and Field name column in red. I checked them in my code, and there is no whitespace or something.

Comment (wy8881) [reply src/main/resources/l10n/JabRef_ko.properties:1852]: Hi, I have already deleted all the translations in the non-English file. I'm not sure what is the reason for labelling the Filename format and Field name column in red. I checked them in my code, and there is no whitespace or something.

File: src/main/java/org/jabref/gui/slr/ManageStudyDefinition.fxml
Comment (koppor) [inline src/main/java/org/jabref/gui/slr/ManageStudyDefinition.fxml:233]: This variable also needs to renamed

File: src/main/java/org/jabref/gui/slr/ManageStudyDefinition.fxml
Comment (koppor) [inline src/main/java/org/jabref/gui/slr/ManageStudyDefinition.fxml:220]: This variable also needs to be renamed

**Review by koppor — CHANGES_REQUESTED** [review]
The PR is a good start. The intention, however, was to go through all the code in the slr code.

Example:

[ManageStudyDefinitionView.java](https://github.com/koppor/jabref/blob/main/src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java#L69-L69)

I double checked the issue description - and it was also contained there. I think, you implemetned the first part. Please also implement the second part:

image

**Comment by wy8881** [comment]
Hi, sorry for that. But I don't sure what is "in SLR code". Is that mean I also change the name of related function?

**Comment by koppor** [comment]
I try to unzip the following comment:

image

The "secret" is to search for "database" in the code:

image

Go through all classes in package `org.jabref.gui.slr`:

image

Finally, you need to modify all classes in package `org.jabref.logic.crawler`:

image

This excercise will teach you how to refactor code in a learning-by-doing way.

**Comment by wy8881** [comment]
Thanks for explanation, I will go and check those code step by step

**Comment by koppor** [comment]
The code changes were done in https://github.com/JabRef/jabref/pull/9989. I think/hope, they are done all.

**Comment by koppor** [comment]
The code changes were integerated in https://github.com/JabRef/jabref/pull/9989 and merged in main. Thank you for your work and timely reaction on the comments!

**Pull request closed without merging**
